### ISSUE TITLE
Ch. 15, Section 15.4.1

### ISSUE THREAD

**Issue by jcornickm** [rafalab/dsbook-part-2#18]
Dear Rafa:  I found this section a bit hard to follow, mostly because in spite of your efforts to explain the basics of baseball to readers not familiar with the game, here I had (initially) no idea of where the 162 came from in the calculaton of pa_per_game, why pa >= 1000 made sense, or what the defensive positions were or what they were called.  Not sure whether it would be worth it to include longer baseball explanations in the chapter, but as a guy from a country where nobody plays this game, I thought I should make you aware of the issue.  Regards

**Comment by rafalab** [comment]
Thanks for the feedback. I plan to rewrite this section to focus more on the statistical concepts and less on the details about baseball.
